cyclonedx-lib/src/temurin/sbom/TemurinGenSBOM.java

```diff
@@ -83,6 +83,8 @@ public static void main(final String[] args) {
                 type = args[++i];
             } else if (args[i].equals("--tool")) {
                 tool =  args[++i];
+            } else if (args[i].equals("--signSBOM")) {
+                cmd = args[++i];
```

Comment:
if `--signSBOM` is a flag (as boolean) should be `cmd = "signSBOM"` which goes to the switch case